src/core/quantum_gate.jl

```diff
@@ -1979,6 +1979,26 @@ Base.inv(gate::RotationY) = RotationY(-gate.theta)
 
 get_gate_parameters(gate::RotationY) = Dict("theta" => gate.theta)
 
+"""
+    rotation_z(target, lambda)
+
+Return a `Gate` that applies a symmetric phase shift of `-lambda/2` and `+lambda/2` to the 
```

Comment:
See discussion on #361 regarding "symmetric phase shift".

Reply:
updated to a more standard docstring